README.md

```diff
@@ -13,19 +15,19 @@ The repo is used to store career-related notes:
 ### Create the repo
 
 1. Click on **Use this template** in top action bar.
-1. For `Name` enter `niteo-career`.
+1. For `Name` enter `niteo-career-<handle>`.
 1. For `Privacy` select `Private`.
 1. Click `Begin import` and wait a few minutes.
 
 ### Create the issues
 
-1. Go to your new `niteo-career` repo.
+1. Go to your new `niteo-career-<handle>` repo.
 1. Click `Issues` tab and then click `Get Started` for `Career Path` item.
 1. Replace dummy data in `[` `]` brackets with your actual values.
 1. Save and repeat for `Catchups`.
 
 ### Grant access
 
-1. Go to your new `niteo-career` repo.
+1. Go to your new `niteo-career-<handle>` repo.
 1. Click `Settings` tab, then click `Collaborators` sidebar item.
-1. Add `@zupo`, `@dmurko`, `@marbeann` and your senior as Collaborators.
+1. Add `@Partners` and your mentor as Collaborators.
```

Comment:
Isn't `senior` more fitting than `mentor`? A mentor can be not your senior.

An example case here is Niko ⇄ Janez.

Reply:
We don't have it defined what a senior is and what a mentor is. I'd wager most people would understand that in-company mentors are involved with their promotions.